crates/proof-of-sql-parser/src/lib.rs

```diff
@@ -9,11 +9,12 @@ extern crate lalrpop_util;
 
 pub mod intermediate_ast;
 
-#[cfg(test)]
+#[cfg(all(test, feature = "parser-test-utility"))]
```

Comment:
Is this change needed? These are just tests, right?
```suggestion
#[cfg(test)]
```